chain/x/oracle/keeper/request_test.go

```diff
@@ -108,9 +109,20 @@ func TestAddPendingResolveList(t *testing.T) {
 }
 
 func TestGetRandomValidatorsSuccess(t *testing.T) {
-	// TODO: Update this test once GetRandomValidators is actually random
+	_, ctx, k := createTestInput()
```

Comment:
Can you test by set block header of context after random will get validator different